source/mainnet/smart-contracts/tutorials/nft-marketplace/index.rst

```diff
@@ -6,11 +6,11 @@ NFT marketplace smart contract tutorial
 
 In this tutorial you will create an NFT Marketplace. The `contract <https://github.com/chainorders/concordium-nft-tutorials>`_ provides some functionalities, such as minting semi-fungible, and non-fungible tokens, buying and selling NFTs with fixed prices, setting commissions for your marketplace, and setting royalties for your NFTs to get some fees from secondary sales in this marketplace. Before you start, it is important to note that this is not a beginner-level tutorial; it does not cover the basics like downloads, node configurations, wallet setup and export, and so on.
 
-This tutorial will be the first part of the Marketplace. There will be no UI interaction. You will invoke the functions with ``concordium-client``. In the end, while implementing a dApp, what you have to do is implement a client that connects your backend with the blockchain. ``concordium-client`` is that client and does that. In the second part which will be released soon, you will implement this marketplace from scratch with an empty React template.
+This tutorial will be the first part of the Marketplace. There will be no UI interaction, so you will have to invoke the functions with ``concordium-client``. In the end, while implementing a dApp, what you have to do is implement a client that connects your backend with the blockchain. ``concordium-client`` is that client and does that.
```

Comment:
It really does not seem like it will be release soon, are there any concrete plans for this?

Reply:
We are not actively expanding the tutorial, so it makes sense that we remove that sentence for now.